legal-api/src/legal_api/models/filing.py

```diff
@@ -68,6 +68,16 @@ class Source(Enum):
                 'CP': 'AFDVT'
             }
         },
+        'agmExtension': {
+            'name': 'agmExtension',
+            'title': 'AGM Extension',
+            'codes': {
+                'BC': 'AGMDT',
+                'BEN': 'AGMDT',
+                'ULC': 'AGMDT',
+                'CC': 'AGMDT'
+            }
+        },
```

Comment:
We can also add a corresponding entry to legal_api/core/meta/filing.py.

I missed this when you worked on agm location change.  Something similar to [this](https://github.com/bcgov/lear/blob/ddcdf621431e5fd3a67182a2c7caac16e690d850/legal-api/src/legal_api/core/meta/filing.py#L85-L95).